packages/jest-types/src/Global.ts

```diff
@@ -119,3 +119,7 @@ export interface Global
     Omit<NodeJS.Global, keyof GlobalAdditions> {
   [extras: string]: unknown;
 }
+
+declare global {
```

Comment:
will this pollute the global env of consumers? if "yes' I'd prefer to not do that